### DISCUSSION TITLE
新手提问：请问readme里面所指的编译，在window下应该怎么编译呢？会报出以下问题

### DISCUSSION THREAD

**Discussion by vleyong** [apache/doris#36372] [A - General / Q&A]
https://github.com/apache/doris-flink-connector：
**How to Build**
You need to copy customer_env.sh.tpl to customer_env.sh before build and you need to configure it before build.
```
git clone git@github.com:apache/doris-flink-connector.git
cd doris-flink-connector/flink-doris-connector
./build.sh
```
**请教三个问题：**
### 2、似乎我直接从github拉取代码到idea，test文件下的类直接run会报错：
> java: 找不到符号
>   符号:   类 DorisReadOptions
>   位置: 程序包 org.apache.doris.flink.cfg
   这是缺少什么依赖吗？应该怎么解决。
    
### 3、请问这个项目有pom文件为什么不被IDEA识别成Maven项目，还是说此项目本身就不是maven项目？
![1718595255941](https://github.com/apache/doris/assets/16420741/a47522a6-23ac-4c51-9f1b-02b564bf5716)

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题

**Comment by zclllyybb** [comment]
You can build in WSL2 in Windows. For native Windows, AFAIK it's not supported.
